test/ExecuteOffer.t.sol

```diff
@@ -81,4 +81,19 @@ contract ExecuteOfferTest is BaseTest {
         // validate that the receiver items are swapped
         assertOfferItemsOwnership(offer.receiverItems.items, offer.sender);
     }
+
+    function testCanExecuteOfferMultiTokens() public {
+        Offer memory offer = _executeMultiTokensOffer();
+        bytes32 offerId = generateOfferId(offer);
+
+        (address sender,,,,,) = echo.offers(offerId);
+
+        // @dev When the offer is executed, it's also deleted
+        assertEq(sender, address(0));
+
+        // validate that the sender items are swapped
+        assertOfferItemsOwnership(offer.senderItems.items, offer.receiver);
+        // validate that the receiver items are swapped
+        assertOfferItemsOwnership(offer.receiverItems.items, offer.sender);
```

Comment:
same thing I guess we should assert offer state in these tests

Reply:
see other comment.